Docs: "Platform integration" - "Writing platform-specific code" doesn't compile in Swift 5

I followed the section on ["Writing platform-specific code"](https://flutter.dev/docs/development/platform-integration/platform-channels). The code presented there for reading the battery level in Swift (Step 5b) doesn't compile in Swift 5:
- There's a typo (`bynaryMessenger` instead of `binaryMessenger`).
- A class name has changed &mdash; from `UIDeviceBatteryState` to `UIDevice.BatteryState`.

Please note that in Xcode, I updated the Flutter project to Swift 5.

Here's the error I got with the current code from the site:

````
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/karsten/workspaces/me/services/company-c/apps/ca_test_flutter/ios/Runner/AppDelegate.swift:22:64: error: value of type 'FlutterViewController' has no member 'bynaryMessenger'; did you mean 'binaryMessenger'?
                                                  binaryMessenger: controller.bynaryMessenger)
                                                                   ^~~~~~~~~~ ~~~~~~~~~~~~~~~
                                                                              binaryMessenger
    Flutter.FlutterViewController:26:14: note: 'binaryMessenger' declared here
        open var binaryMessenger: FlutterBinaryMessenger! { get }
                 ^
    /Users/karsten/workspaces/me/services/company-c/apps/ca_test_flutter/ios/Runner/AppDelegate.swift:39:31: error: 'UIDeviceBatteryState' has been renamed to 'UIDevice.BatteryState'
        if device.batteryState == UIDeviceBatteryState.unknown {
                                  ^~~~~~~~~~~~~~~~~~~~
                                  UIDevice.BatteryState
    UIKit.UIDeviceBatteryState:2:18: note: 'UIDeviceBatteryState' was obsoleted in Swift 4.2
    public typealias UIDeviceBatteryState = UIDevice.BatteryState
                     ^
    note: Using new build systemnote: Planning buildnote: Constructing build description

Could not build the application for the simulator.
Error launching application on iPhone 5s.
````

To fix this, lines 8 & 9 need to be replaced:

````
let batteryChannel = FlutterMethodChannel(name: "com.betterprojectsfaster.test.ca.flutter/battery",
                                              binaryMessenger: controller.binaryMessenger)
````

The second code sample needs this line 4:

````
if device.batteryState == UIDevice.BatteryState.unknown {
````

Here's my environment:
- macOS 10.14.6
- Project created with Flutter 1.7
- Xcode 10.3

--- a/src/docs/development/platform-integration/platform-channels.md
+++ b/src/docs/development/platform-integration/platform-channels.md
@@ -575,7 +575,7 @@ a `FlutterMethodChannel` tied to the channel name
 @objc class AppDelegate: FlutterAppDelegate {
   override func application(
     _ application: UIApplication,
-    didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
 
     let controller : FlutterViewController = window?.rootViewController as! FlutterViewController
     let batteryChannel = FlutterMethodChannel(name: "samples.flutter.dev/battery",
@@ -602,7 +602,7 @@ Add the following as a new method at the bottom of `AppDelegate.swift`:
 private func receiveBatteryLevel(result: FlutterResult) {
   let device = UIDevice.current
   device.isBatteryMonitoringEnabled = true
-  if device.batteryState == UIDeviceBatteryState.unknown {
+  if device.batteryState == UIDevice.BatteryState.unknown {
     result(FlutterError(code: "UNAVAILABLE",
                         message: "Battery info unavailable",
                         details: nil))

